practice-two/src/style/sections/review.css

```diff
@@ -0,0 +1,158 @@
+.review {
+  padding-top: 28px;
+  padding-bottom: 48px;
+}
+
+.review__wrapper {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.review__card.review__card--light {
+  display: none;
+}
+
+.review__card {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 240px;
+  height: 280px;
+  border-radius: 5px;
+  box-shadow: 0px 5px 15px rgba(0, 0, 0, 0.4);
```

Comment:
I see that you have use this color many time, please consider to create variable for it